statistics/update_test.go

```diff
@@ -269,6 +270,11 @@ func (s *testStatsUpdateSuite) TestAutoUpdate(c *C) {
 	testKit.MustExec("use test")
 	testKit.MustExec("create table t (a int)")
 
+	statistics.AutoAnalyzeMinCnt = 0
+	defer func() {
+		statistics.AutoAnalyzeMinCnt = 1000
```

Comment:
1000 is too small?

Reply:
I think 1000 is a conservative value. If it works well in production, we can scale it up.